### PR TITLE
Dorska/analytics overlay

### LINKED ISSUE #551: Overlay Video Analytics on Video Player
Per discussion w/ @pdpinch on 2018-04-13.

"As a user with access to analytics, I want to know how interesting analytics moments match up to video moments."

One way to address this want is to overlay an analytics chart on top of a playing video.

The analytics chart could be interactive, so that clicking the chart seeks to the corresponding spot in the video.
Or the chart could simply be the same width as the video player's "seek bar", and positioned near it, so that a user can match up events w/ the video player seek bar.

I estimate this would take 4-5 developer days to implement.

### PR DESCRIPTION
**Pull request by adorsk**
#### What are the relevant tickets?
Closes #551 .

#### What's this PR do?
Displays analytics as an overlay on top of the video.

#### How should this be manually tested?
1. Find a video for which you have (a) analytics data, and (b) edit access .
2. Go to the detail page for the video.
3. Click the `show/hide analytics` button. An overlay should pop up over the video.
4. The overlay should show a loading message at first.
5. The overlay should then show an analytics chart with a time slider.
6. Click on the time slider: the video time should change, and the analytics info table should show data for the updated time.
7. Start playing the video. The analytics overlay should update as the video time changes.
8. Click the `show/hide analytics` button again. The overlay should disappear.

Confirm that the chart and table look reasonable for:
1. A video with singlecam analytics
2. A video with multicam analytics.

### CODE REVIEW

**Comment by adorsk** [comment]
@mbertrand ah, sorry, I should have mentioned that the table tab was intentionally removed. The analytics dialogs have also been removed.

**Comment by adorsk** [comment]
@mbertrand I decided to add the legend back in (for multicam only), because otherwise it wasn't clear what the colors mean. So, the legend is intentional.

**Comment by adorsk** [comment]
Nice catch, that's definitely a bug. I'll look into, will update when I fix
it.

File: static/js/containers/VideoDetailPage.js [hunk 1]
```diff
@@ -222,6 +199,82 @@ class VideoDetailPage extends React.Component<*, void> {
...
+          analyticsOverlayIsVisible={videoUi.analyticsOverlayIsVisible}
+        />
+        {videoUi.analyticsOverlayIsVisible
+          ? this.renderAnalyticsOverlay()
+          : null}
+      </div>
+    )
+  }
+
+  renderAnalyticsOverlay() {
```
Comment (mbertrand) [inline static/js/containers/VideoDetailPage.js]: Codecov indicates that this function isn't covered, could you add a unit test for it? And if possible/necessary, additional unit tests [to reach 95% coverage](https://codecov.io/gh/mitodl/odl-video-service/pull/563/diff).

Comment (mbertrand) [reply static/js/containers/VideoDetailPage.js]: PS There are also conflicts with the master branch that need to be resolved.

**Comment by adorsk** [comment]
@mbertrand I think the conflicts were resolved (?) If you're still seeing them, let me know (I'm currently not seeing them).

**Comment by mbertrand** [comment]
@adorsk Still seeing conflicts, on this page at the bottom, just under "All checks have passed (4 successful checks)"

**Pull request merged**